### DISCUSSION TITLE
log configuration want to log ping & pong

### DISCUSSION THREAD

**Discussion by kingter-sutjiadi** [inspircd/inspircd#2130] [Support] [answered]
Hi, i'm using this configuration 

```
<log method="file"
     level="debug"
     type="* -USERINPUT"
     flush="100"
     target="inspircd.log">
```

but when i tail the log file somehow i don't see any ping or pong command logged on the log file 

am i missing something?

Thank you

**Comment by genius3000** [comment]
`flush="100"` means it will buffer 100 lines in memory and then write them to the log file. Try setting it to 1.

**Reply by kingter-sutjiadi** [reply]
tried it still no sign of PONG  🙏

**Reply by sadiepowell** [reply, marked as answer]
Raw messages are logged at the rawio level not the debug level.

**Reply by kingter-sutjiadi** [reply]
thank you @SadieCat 🙏
